model_runner/validator/_tests/test_validator.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from pydantic import ValidationError
+
+from model_runner.validator.validator_utils import ConfigModel
+
+
+@pytest.fixture(scope="session")
+def base_config(tmp_path_factory):
+    my_file = tmp_path_factory.mktemp("data") / "myfile"
+    my_file.touch()
+
+    config = {
+        "data": tmp_path_factory._basetemp,
```

Comment:
I am wondering if this should be part of the "hyperparameters" so it is possible to iterate over different datasets (e.g., for cross validation)